prac_09/unreliable_car.py

```diff
@@ -0,0 +1,27 @@
+"""Prac 09 - Unreliable Car"""
+
+from prac_09.car import Car
+from random import randint
+
+
+class UnreliableCar(Car):
+    """Initialise an unreliable car"""
+
+    def __init__(self, name, fuel, reliability):
+        super().__init__(name, fuel)
+        self.reliability = float(reliability)
+
+    def drive(self, distance):
+        """Override drive method so unreliable car only drives
+         when reliability is less than a random number"""
+        if randint(0, 100) < self.reliability:
```

Comment:
For the following section, you could have it so that if the car goes, it does something like "distance = super().drive(distance)" otherwise, it prints "Car failed :(" (this would also save you that step of accounting for if the distance is greater than the remaining fuel)